### ISSUE TITLE
question: I followed directions on user/group set up - can not restore outside of crashplan appdata folder

### ISSUE THREAD

**Issue by sstretchh** [jlesage/docker-crashplan-pro#346]
I am using UNRAID/DOCKER

I created a user called crashplan. I put the UID/GID where it is supposed to go. I gave this user permissions to ALL shares write/read with in unraid

everything is backing up with NO issues, working great. 

I am testing restoring when I restore, I can only restore to the appdata folder where crashplan files exist. I want to pick a destination within one of the file shares, is this possible? 

do I need to somehow push the permissions onto the file shares even though the new user already has read/write, or should I turn on NFS?

**Comment by sstretchh** [comment]
after reading more, it looks like when storage is mounted, its read-only by design that I need to pass an argument to "$HOME:/storage:rw"

I just need to figure that out, I am wondering if I can just map an addition share called restored.

**Comment by jlesage** [comment]
Yes, the template in unRAID is setting `/storage` as read-only be default.
If you want to restore to the same location, you can edit the container settings and change `/storage` to be read/write.
You can also add a new `Path`, configured with a read/write access, that will be used as the location where to restore your files.

**Comment by sstretchh** [comment]
I know this is not UNRAID support, but how would I change that within UNRAID ?

I will try testing adding a new path, and see how that goes also. If I add the same path, should I just blank out the path that is predefined on /storage with in the unraid template ?

**Comment by jlesage** [comment]
If you add a new path, it must be different from the /storage one.

For example, you can add a path that maps `/mnt/user/restore` on the host (unRAID) to `/restore` in the container.  Then, in CrashPlan, when you restore files, you select `/restore` as the location where to restore the files.
